src/core/main.c

```diff
@@ -731,6 +737,16 @@ static void set_manager_defaults(Manager *m) {
         m->default_oom_score_adjust_set = arg_default_oom_score_adjust_set;
         m->default_oom_score_adjust = arg_default_oom_score_adjust;
 
+#ifdef SMACK_DEFAULT_PROCESS_LABEL
+        if (!arg_default_smack_process_label)
+                m->default_smack_process_label = SMACK_DEFAULT_PROCESS_LABEL;
+        else
+#endif
+        if (streq(arg_default_smack_process_label, "/"))
+                m->default_smack_process_label = NULL;
+        else
+                m->default_smack_process_label = arg_default_smack_process_label;
+
```

Comment:
hmm, I think it would be better to move this into a function `manager_set_default_smack_process_label()`, and hide the default fallback stuff there.